server/api_archive_integration_test.go

```diff
@@ -209,45 +209,40 @@ func ServerWithArchiveFiles(name string) (*GCAServer, string, error) {
 		return nil, "", err
 	}
 
-	// Generate a keypair for a device.
-	authPub, authPriv := glow.GenerateKeyPair()
-	auth := glow.EquipmentAuthorization{ShortID: 0, PublicKey: authPub}
-	sb := auth.SigningBytes()
-	sig := glow.Sign(sb, gcaPrivKey)
-	auth.Signature = sig
-	_, err = gcas.saveEquipment(auth)
+	// Submit new hardware via API
+	auth, authPriv, err := gcas.submitNewHardware(0, gcaPrivKey)
```

Comment:
does this need to be called `staticSubmitNewHardware`?

Reply:
when writing test libraries I often forget that test functions need to follow the same concurrency conventions as production code

Reply:
no I looked at the implementation, it uses locks so it needs to be called `managedSubmitNewHardware`

Reply:
we can do that in a follow-up